testing/tests/DevExpress.ui.widgets.dataGrid/columnsView.tests.js

```diff
@@ -516,7 +516,13 @@ QUnit.module('API methods', {
 
         this.option('cellHintEnabled', true);
         const $table = $(this.columnsView._createTable());
-        $container.html($('<div class = \'dx-datagrid-rowsview dx-datagrid-nowrap\' />').append($table.append(this.columnsView._createColGroup(this.columns), '<tr class="dx-row dx-master-detail-row"><td><div id="content" style="overflow: hidden;"><div style="width: 600px; height: 30px;">Test</div></div></td></tr>')));
+        $container.html($('<div class = \'dx-datagrid-rowsview dx-datagrid-nowrap\' />').append($table.append(this.columnsView._createColGroup(this.columns), '<tr class="dx-row dx-master-detail-row"><td><div id="content"><div>Test</div></div></td></tr>')));
+        $container.find('#content').css('overflow', 'hidden');
+
+        $container.find('#content > div').css({
+            width: '600px',
+            height: '30px'
+        });
```

Comment:
Let's add more obvious formatting here:

```suggestion
        $container.html($(`<div class = 'dx-datagrid-rowsview dx-datagrid-nowrap'/>`)
        .append(
          $table.append(
            this.columnsView._createColGroup(this.columns),
            `<tr class="dx-row dx-master-detail-row">
              <td>
                <div id="content">
                  <div container-test-text="true">Test</div>
                </div>
              </td>
            </tr>`
           )));
           
        $container.find('#content').css('overflow', 'hidden');
        $container.find('#content > [container-test-text]').css({
            width: '600px',
            height: '30px'
        });
```